File: AspNetCore_gdr2333_Front+Back/Back/Types/DataBase/Capcha.fs
namespace Back.Types.DataBase

open System
open System.ComponentModel.DataAnnotations

type Capcha(uuid: string, result: string) =
    class
        [<Key>]
        member val Uuid = uuid with get

        member val Result = result with get
        member val DeleteAt = DateTime.Now.AddHours 1 with get
    end

File: AspNetCore_gdr2333_Front+Back/Back/Types/DataBase/Capcha.cs
using System.ComponentModel.DataAnnotations;

namespace Back.Types.DataBase;

public class Capcha
{
    [Key]
    public string Guid { get; set; }

    public string Result { get; set; }

    public DateTime DeleteAt { get; set; }

    public Capcha(string guid, string result)
    {
        Guid = guid;
        Result = result;
        DeleteAt = DateTime.Now.AddHours(1);
    }

    public Capcha()
    {
    }
}
